docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -146,6 +146,11 @@ api:
   # CLI flag: -distributor.enable-otlp-metadata-storage
   [enable_otel_metadata_translation: <boolean> | default = false]
 
+  # (experimental) Enable routes to support the migration and operation of the
+  # Grafana Alertmanager.
+  # CLI flag: -api.experimental-grafana-alertmanager-routes-enabled
+  [experimental_grafana_alertmanager_routes_enabled: <boolean> | default = false]
```

Comment:
did you consider adding this to the alertmanager block? Perhaps something like `-alertmanager.grafana-alertmanager-routes-enabled`? Also removing experimental since the flag might move out of being experimental at some point. The convention in Mimir is to only tag the flag as experimental (as you're done), but not include it in the name of the flag.

Also do you expect there will be other behavioural changes anticipated when the Grafana AM is supported? Asking whether it won't be more appropriate fur future use-cases to call this `-alertmanager.grafana-alertmanager-compatibility-enabled` for example.

_I lack a bit of context on this endeavour, so these suggestions may be out of place_

Reply:
Thanks for this - I saw another example that included the word "experimental" (https://github.com/grafana/mimir/pull/7057) but on a second look, it appears to be used noun as opposed to a verb so you're correct.

Reply:
I took a stab at this in [083abea](https://github.com/grafana/mimir/pull/6682/commits/083abea0147e15612272e7bfab850550f649e3db) - what do you think?